vortex-file/src/read/mask.rs

```diff
@@ -105,29 +128,35 @@ impl RowMask {
                 return Err(err());
             }
 
-            let mut bitmap = Bitmap::new();
+            let mut buffer_builder = BooleanBufferBuilder::new_from_buffer(MutableBuffer::new((end - begin).div_ceil(8)), end - begin);
```

Comment:
`BooleanBufferBuilder::new(end-begin)` will achieve the same effect

Reply:
Not quite, the code here was wrong